core/domain/services/graphql/resolvers/FieldResolver.php

```diff
@@ -136,6 +138,25 @@ public function resolveId($source)
     }
 
 
+    /**
+     * Resolve the cache ID
+     *
+     * @param mixed     $source
+     * @return string
+     * @since $VID:$
+     */
+    public function resolveCacheId($source)
+    {
+        $model_name = $this->model->item_name();
+        $ID         = $source->ID();
+        $uniqid     = uniqid();
+        // Since cacheId does not need to be globally unique
+        // $uniqid is sufficient, still adding the model name and ID
+        // may be we need/use them in future.
+        return "{$model_name}:{$ID}:{$uniqid}";
+    }
```

Comment:
you can also pass a prefix to `uniqid()` for the first parameter and it's always advisable to set the second parameter to true, which increases the amount of randomness

```suggestion
    public function resolveCacheId($source)
    {
        $model_name = $this->model->item_name();
        $ID         = $source->ID();
        // Since cacheId does not need to be globally unique
        // $uniqid is sufficient, still adding the model name and ID
        // may be we need/use them in future.
        return uniqid("{$model_name}:{$ID}:", true);
    }
```